### ISSUE TITLE
Resnet 18 ran for a while and stopped for a while

### ISSUE THREAD

**Issue by 408550969** [vgenty/resnet18#1]
After run a period of time it will be suspended for some time, the CPU usage is 0.（I use the CPU mode） Is it all right? Why does this happen？
And why did not display accuracy？
I0608 19:46:13.666218 32988 solver.cpp:219] Iteration 0 (-5.60519e-045 iter/s, 3.577s/100 iters), loss = 1.94591
I0608 19:46:13.667219 32988 solver.cpp:238]     Train net output #0: loss = 1.94591 (* 1 = 1.94591 loss)
I0608 19:46:13.668220 32988 sgd_solver.cpp:105] Iteration 0, lr = 0.1
I0608 19:48:57.167953 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 19:51:58.403479 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 19:52:17.776361 32988 solver.cpp:219] Iteration 100 (0.274643 iter/s, 364.109s/100 iters), loss = 1.60979
I0608 19:52:17.776361 32988 solver.cpp:238]     Train net output #0: loss = 1.6098 (* 1 = 1.6098 loss)
I0608 19:52:17.777361 32988 sgd_solver.cpp:105] Iteration 100, lr = 0.1
I0608 19:54:56.657876 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 19:57:58.029636 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 19:58:14.851822 32988 solver.cpp:219] Iteration 200 (0.280053 iter/s, 357.075s/100 iters), loss = 1.60626
I0608 19:58:14.851822 32988 solver.cpp:238]     Train net output #0: loss = 1.60627 (* 1 = 1.60627 loss)
I0608 19:58:14.852823 32988 sgd_solver.cpp:105] Iteration 200, lr = 0.1
I0608 20:00:49.581725 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:03:42.521733 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:04:00.139472 32988 solver.cpp:219] Iteration 300 (0.289614 iter/s, 345.287s/100 iters), loss = 1.60507
I0608 20:04:00.140473 32988 solver.cpp:238]     Train net output #0: loss = 1.60508 (* 1 = 1.60508 loss)
I0608 20:04:00.141474 32988 sgd_solver.cpp:105] Iteration 300, lr = 0.1
I0608 20:06:44.129534 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:09:34.286691 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:09:52.100536 32988 solver.cpp:219] Iteration 400 (0.284123 iter/s, 351.96s/100 iters), loss = 1.60457
I0608 20:09:52.101536 32988 solver.cpp:238]     Train net output #0: loss = 1.60457 (* 1 = 1.60457 loss)
I0608 20:09:52.102537 32988 sgd_solver.cpp:105] Iteration 400, lr = 0.1
I0608 20:12:40.149827 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:15:41.505817 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:15:59.269629 32988 solver.cpp:219] Iteration 500 (0.272354 iter/s, 367.169s/100 iters), loss = 1.60434
I0608 20:15:59.269629 32988 solver.cpp:238]     Train net output #0: loss = 1.60434 (* 1 = 1.60434 loss)
I0608 20:15:59.270629 32988 sgd_solver.cpp:105] Iteration 500, lr = 0.1
I0608 20:18:42.699316 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:18:52.792026 32988 blocking_queue.cpp:49] Waiting for data
I0608 20:26:10.025845 31672 data_layer.cpp:73] Restarting data prefetching from start.
I0608 20:26:57.990689 32988 solver.cpp:219] Iteration 600 (0.151809 iter/s, 658.721s/100 iters), loss = 1.60423
I0608 20:26:57.991689 32988 solver.cpp:238]     Train net output #0: loss = 1.60423 (* 1 = 1.60423 loss)
I0608 20:26:57.991689 32988 sgd_solver.cpp:105] Iteration 600, lr = 0.1
